parsec/backend/http.py

```diff
@@ -367,33 +367,51 @@ async def _api_check_auth_and_load_body(self, req: HTTPRequest, req_serializer):
         return None, data
 
     async def _http_api_anonymous(self, req: HTTPRequest, **kwargs: str) -> HTTPResponse:
+        # Check whether the organization exists
```

Comment:
Are we sure this is what we want to do ?

On the one hand it's tempting to have bootstrap as part of the anonymous family given they all refer to an organization_id, but on the other hand the fact the organization should not be bootstrapped (and may even not be created at all) make it very special (hence those non-trivial changes that may be error prone...)

I don't have a strong opinion, but I think it's interesting to think a bit about that before choosing our poison ;-)

Reply:
Agreed, as a first implementation I chose to follow what was already done here:
https://github.com/Scille/parsec-cloud/blob/f4a624d2c0152842f4cd5d82b510728d85c3428a/parsec/backend/handshake.py#L194-L235

But we might want to change that, I don't have a strong opinion either.

Reply:
> But we might want to change that, I don't have a strong opinion either.

So how do we decide if it's good or not ? :trollface:

Reply:
> the fact the organization should not be bootstrapped (and may even not be created at all) make it very special (hence those non-trivial changes that may be error prone...)

The fact that the organization should not be bootsrapped is already managed correctly in the command itself. The weird part is the spontaneous creation of the organization if the backend configuration allows it.

Reply:
Let's keep it this way for the moment and investigate later how we want to evolve this command.